ADAL/src/ui/mac/ADNTLMUIPrompt.m

```diff
@@ -42,13 +42,14 @@ + (void)presentPrompt:(void (^)(NSString * username, NSString * password))comple
     [alert addButtonWithTitle:NSLocalizedString(@"Cancel", nil)];
     
     ADCredentialCollectionController* view = [ADCredentialCollectionController new];
+    SAFE_ARC_AUTORELEASE(view);
     [view.usernameLabel setStringValue:NSLocalizedString(@"User Name", nil)];
     [view.passwordLabel setStringValue:NSLocalizedString(@"Password", nil)];
     [alert setAccessoryView:view.customView];
     
     [alert beginSheetModalForWindow:[NSApp keyWindow] completionHandler:^(NSModalResponse returnCode)
     {
-        if (returnCode == 1)
+        if (returnCode == 1000)
         {
```

Comment:
I don't know why previously it is "1" for the returnCode? :P For now, return code is 1000 if the first button is clicked.